src/ENVPAK.CPP

```diff
@@ -7,9 +7,16 @@
 /*------------------------------- INCLUDES --------------------------------*/
 #include "cnglob.h"
 // #include "cse.h"
+#if !defined( ELIMINATE_WINDOWS)
+#include <windows.h>
+#include <direct.h>	// getcwd
+#endif
+
+#ifdef linux
```

Comment:
`linux` is deprecated. Use `__linux__` instead.

Reply:
Ok, good to know for the future.